Gir.toml

```diff
@@ -156,6 +161,7 @@ generate = [
     "Gtk.Separator",
     "Gtk.SeparatorMenuItem",
     "Gtk.SeparatorToolItem",
+    "Gtk.Settings",
```

Comment:
Someone might need it eventually I guess. But it feels like a hack, which probably isn't cross-platform. And we don't support properties yet so it'd be of limited use anyway.